doc/nrf/known_issues.rst

```diff
@@ -851,7 +852,7 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
-.. rst-class:: v2-0-2
+.. rst-class:: v2-1-0 v2-0-2
```

Comment:
Actually, we decided to close this issue, because it's no longer visible after upmerging Matter in NCS. So it's resolved in NCS 2.1.0.

Reply:
Updated.